libsignal-service/examples/storage.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::diverging_sub_expression)]
+
```

Comment:
Can you annotate the section where this is needed, instead of a global allow? If not, can you write a comment why this is here?